### DISCUSSION TITLE
Meeting 2: January 19th 2023 @ 15:00 UTC

### DISCUSSION THREAD

**Discussion by ml-evs** [marda-alliance/metadata_extractors#11] [Announcements]
Our next meeting will run on [January 19th at 15:00 UTC/16:00 CET/10:00 EST](https://www.timeanddate.com/worldclock/fixedtime.html?iso=20230119T15) on Jitsi at https://meet.jit.si/marda_extractors.

All are welcome!

**Comment by dwinston** [comment]
hello? I'm in https://meet.jit.si/marda_extractors

**Comment by dwinston** [comment]
damn it. *January* 19th. I need more coffee

**Reply by ml-evs** [reply]
Time for a Christmas holiday, methinks :upside_down_face: 

I can't remember if this was a time you could make @dwinston, sorry if it's not! I'd be happy to meet individually and catch you up on the project if you are interested.

**Reply by dwinston** [reply]
I can make it. I have moved my calendar entry to the correct date. 😄

**Reply by WardLT** [reply]
Me too! Looking forward to meeting y'all

**Comment by ml-evs** [comment]
Just an additional reminder (for @dwinston's sake!) that this is tomorrow at 15:00 UTC (22 hours and 10 mins from the time of this comment)

**Reply by dwinston** [reply]
Unfortunately, the state of New York has insisted on extracting jury duty from me during that time. Alas. Third time will be a charm?

**Reply by ml-evs** [reply]
Hey @dwinston, we're skipping February meetings due to the MaRDA annual meeting (there will be a presentation from us, I guess maybe also one from data dicts?)

If you wanted to catch up on what we're doing here then we've arranged an "office hours" chat on Tuesday next week: https://github.com/marda-alliance/metadata_extractors/discussions/6#discussioncomment-4820016

I guess we'll probably have one more before the MaRDA annual meeting too. Still trying to find a date for the March meeting, but I'm not going to email spam everyone with a survey this time. Here's a link if you want: http://whenisgood.net/d7tr52d

**Comment by ml-evs** [comment]
Minutes with links to the slides etc. will appear shortly in this repo under [`./meetings`](https://github.com/marda-alliance/metadata_extractors/tree/main/meetings). I'll send out an email this afternoon to follow up about the next meetings.
